Cargo.toml

```diff
@@ -36,41 +36,42 @@ path = "src/lib.rs"
 # Dependencies on other LNP/BP repositories
 # -----------------------------------------
 amplify = { version = "~3.13.0", features = ["apfloat"] }
-bp-core = { version = "~0.8.0" }
-lnpbp = { version = "~0.8.0", features = ["zip"] }
-stens = "~0.7.1"
-strict_encoding = { version = "~0.8.0", features = ["crypto", "chrono", "bitcoin", "float"] }
-commit_verify = { version = "~0.8.0", features = ["rand", "bulletproofs"] }
-descriptor-wallet = "~0.8.0"
-aluvm = { version = "~0.7.0", features = ["std", "strict_encoding"] }
+bp-core = { version = "~0.9.0-rc.1" }
+lnpbp = { version = "~0.9.0-rc.1", features = ["zip"] }
+stens = "~0.9.0-rc.1"
+strict_encoding = { version = "~0.9.0-rc.1", features = ["crypto", "chrono", "bitcoin", "float"] }
+commit_verify = { version = "~0.9.0-rc.1", features = ["rand", "bulletproofs"] }
+descriptor-wallet = "~0.9.0-rc.1"
+aluvm = { version = "~0.9.0-rc.1", features = ["std", "strict_encoding"] }
 # Dependencies on core rust-bitcoin ecosystem projects
 # ----------------------------------------------------
-bitcoin = { version = "~0.28.1", features = ["rand"] }
+bitcoin = { version = "~0.29.2", features = ["rand"] }
 # Core rust projects
 # ------------------
 #   This strange naming is a workaround for cargo inability to define required
 #   features for a dependency. See
 #   <https://github.com/rust-lang/api-guidelines/issues/180> for the explanation
 #   and references.
-half = "~1.8.2"
+half = "~2.1.0"
 serde_crate = { package = "serde", version = "1", features = ["derive"], optional = true }
-serde_with = { version = "1.8", features = ["hex"], optional = true }
+serde_with = { version = "1.14", features = ["hex"], optional = true }
 once_cell = "1.12.0"
 # Other dependencies:
 # ------------------------------
 #    We are of no control of what's happening in Grin and would like to prevent
 #    updates w/o checking them manually. Should Elements Project secpk256k-zkp
 #    accept bulletproofs code, we will switch to it
-grin_secp256k1zkp = { version = "=0.7.11" }
```

Comment:
GitHub ate my comment earlier... The concern I meant to raise, the code in this repo appears quite old:
https://github.com/BP-WG/rust-secp256k1
That could cause problems when considering WASM compatiblity. Something to be aware of.

Reply:
It is not that repo. https://github.com/BP-WG/rust-secp256k1 is just a fork of normal `secp256k1` for doing PR to that repo; we do not use, release or maintain it.

The one replaced here is github.com/LNP-BP/rust-secp256k1-zkp which I forked from mimbewimble, removed unneeded stuff, updated dependencies etc.

Reply:
Gotcha... I've created a PR to help avoid that confusion.